README.md

```diff
@@ -39,7 +39,7 @@ For beginners, you can use a free service like ntfy.sh (do consider donating if
 
 ## Web Server
 
-MollySocket exposes a web server so that Molly can send the information it needs to operate. You should configure TLS with a reverse proxy in front of MollySocket.
+MollySocket exposes a web server so that Molly can send the information it needs to operate. You should configure TLS with a reverse proxy in front of MollySocket. Molly can only connect to the proxy server over HTTPS.
```

Comment:
```suggestion
MollySocket exposes a web server so that Molly can send the information it needs to operate. You must configure TLS with a reverse proxy in front of MollySocket. Molly can only connect to the server over HTTPS.
```